controllers/deviceprofile_syncer.go

```diff
@@ -2,43 +2,51 @@ package controllers
 
 import (
 	"context"
+	"fmt"
```

Comment:
please add license header for all files.

Reply:
done